draft-irtf-cfrg-vdaf.md

```diff
@@ -3240,6 +3252,21 @@ the inputs.  It is possible that a future VDAF could incorporate differential
 privacy features, e.g., by injecting noise before the sharding stage and
 removing it after unsharding.
 
+## Requirements for the Nonce
+
+The sharding and preparation steps of VDAF execution depend on a nonce
+associated with the Client's report. To ensure privacy of the underlying
+measurement, the Client MUST generate this nonce using a CSPRNG. This is
+required in order to leverage security analysis for the privacy definition of
+{{DPRS23}}, which assumes the nonce is chosen at random prior to generating the
+report.
+
+Other security considerations may require the nonce to be non-repeating. For
+example, to achieve differential privacy it is necessary to avoid "over
+exposing" a measurement by including it too many times in a single batch or
+across multiple batches. It is RECOMMENDED that the nonce generated by the
+Client be used for replay protection.
```

Comment:
Small clarification:
```suggestion
Other security considerations may require the nonce to be non-repeating. For
example, to achieve differential privacy it is necessary to avoid "over
exposing" a measurement by including it too many times in a single batch or
across multiple batches. It is RECOMMENDED that the nonce generated by the
Client be used by Aggregators for replay protection.
```